api/src/main/java/com/example/api/service/util/EmailService.java

```diff
@@ -0,0 +1,56 @@
+package com.example.api.service.util;
+
+import com.example.api.model.user.User;
+import com.mailjet.client.ClientOptions;
+import com.mailjet.client.MailjetClient;
+import com.mailjet.client.MailjetRequest;
+import com.mailjet.client.MailjetResponse;
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.errors.MailjetSocketTimeoutException;
+import com.mailjet.client.resource.Emailv31;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+@Transactional
+public class EmailService {
+    private final String API_KEY = "";
+    private final String SECRET_API_KEY = "";
```

Comment:
Mam tutaj rozkminę. Może dodać w resource'ach projektu plik z envami, dodać do niego pole EMAIL_API_KEY i na razie wstawić tam pusty string. A tutaj w kodzie brać te zmienne. Nie jestem pewien jak to się robi w javie ale w jsie jak dodamy plik `.env.development` i tam wrzucimy zmienne to możemy się do nich odwołać w kodzie jako do zmiennych środowiskowych. Gdy ktoś zechce ustawić klucz to nie będzie grzebał w kodzie tylko w resource'ach.

Reply:
Przeniosłem klucze do application.properties